test/executable_test.dart

```diff
@@ -76,41 +33,110 @@ ProcessResult checkProject(
 
 void main() {
   group('dependency_validator', () {
+    setUp(() async {
+      // Create fake project that any test may use
+      final fakeProjectPubspec = ''
+          'name: fake_project\n'
+          'version: 0.0.0\n'
+          'private: true\n'
+          'environment:\n'
+          '  sdk: \'>=2.4.0 <3.0.0\'\n'
+          'dev_dependencies:\n'
+          '  dependency_validator:\n'
+          '    path: ${Directory.current.path}\n';
```

Comment:
To avoid having to escape quotes and manually insert the `\n` line breaks, we could copy the `unindent()` util:
https://github.com/Workiva/over_react/blob/315f62be4a8462d841dea31b6563cfb976fa71f3/lib/src/util/string_util.dart#L17-L44

and then use multiline strings like so:

```dart
      final fakeProjectPubspec = unindent('''
          name: fake_project
          version: 0.0.0
          private: true
          environment:
            sdk: '>=2.4.0 <3.0.0'
          dev_dependencies:
            dependency_validator:
              path: ${Directory.current.path}
          ''');
```

Reply:
Ooooh I was looking for something like that!